mro/__init__.py

```diff
@@ -155,59 +131,94 @@ def none_transform(column_default, data_type):
             if column_default:
                 column_default, get_value_on_insert = data_type[2](column_default, column[7])
 
-            code_start = 'data_types.{}("{}", {}, '.format(data_type[0], column_name, column_index)
-            code_end = 'not_null={0}, is_updatable={1}, get_value_on_insert={2}, is_primary_key={3})'.format(not is_nullable, is_updateable, get_value_on_insert, is_primary_key)
-
-            code = data_type[1](column, code_start, code_end)
+            col_data['data_type'] = data_type[0]
+            col_data['column_name'] = column_name
```

Comment:
no longer building the code up as a big string, so removed the old string formatting code and instead passing a dict that we can use to create the tables